src/bin/snapshots/snapshots__ModuleContents.hello-world.input.snap

```diff
@@ -36,43 +36,43 @@ Ok(
                                     )
                                 }
                             ),
-                            _ => Stmt::Expr(
-                                _ => Expr {
-                                    attrs: _ => [],
-                                    kind: _ => ExprKind::Unary {
-                                        expr: _ => Expr {
-                                            attrs: _ => [],
-                                            kind: _ => ExprKind::Paren {
-                                                attrs: _ => [],
-                                                expr: _ => Expr {
-                                                    attrs: _ => [],
-                                                    kind: _ => ExprKind::Literal(_)
+                            Stmt::Expr(
+                                Expr {
+                                    attrs: [],
+                                    kind: ExprKind::Unary {
+                                        expr: Expr {
+                                            attrs: [],
+                                            kind: ExprKind::Paren {
+                                                attrs: [],
+                                                expr: Expr {
+                                                    attrs: [],
+                                                    kind: ExprKind::Literal
                                                 }
                                             } | ExprKind::Tuple {
-                                                attrs: _ => [],
-                                                exprs: _ => [
-                                                    _ => Expr {
-                                                        attrs: _ => [],
-                                                        kind: _ => ExprKind::Literal(_)
+                                                attrs: [],
+                                                exprs: [
+                                                    Expr {
+                                                        attrs: [],
```

Comment:
I wonder whether we should replace `\n *\w+: \[\],\n` with `\n`, to omit empty stuff like this.